ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/point/DropAPinTaskViewModel.kt

```diff
@@ -55,6 +65,8 @@ constructor(resources: Resources, private val uuidGenerator: OfflineUuidGenerato
       id = uuidGenerator.generateUuid(),
       type = FeatureType.USER_POINT.ordinal,
       geometry = point,
+      // TODO: Set correct pin color.
+      style = Feature.Style(pinColor),
```

Comment:
As this is the only usage, should we consider inlining the code? Then we'd have to set the job to a class var in the super class. Thoughts?

```
style = Feature.Style(job.getDefaultColor())
```

Reply:
The superclass is scoped more widely that geometry tasks, and each geometry task may require slightly different styling treatments. Also, I'd prefer to avoid too much indirection to keep things locally comprehensible.

Sorry I missed this TODO, removing it.